test_schemas/en/test_relationships.json

```diff
@@ -678,11 +689,43 @@
                             ]
                         },
                         {
-                            "type": "Confirmation",
-                            "id": "confirmation",
+                            "id": "relationship-interstitial",
                             "content": {
-                                "title": "Thank you for your answers, do you wish to submit"
-                            }
+                                "title": "Relationship section",
```

Comment:
I have had to add an Interstitial and change the confirmation into a summary page into this test. The reason was the test was jumping to a block multiple back by using the url, effectively simulating using previous. However as we have changed how previous works, and after the use of previous the respondent follows their path back, some of the tests associated with this will fail, as they are expecting a jump to the end. If instead we use a summary click which now is the only way to have that behaviour this will work as expected. The Interstitial went in as there was a test where the behaviour of a block after a relationship collector was being tested.